Documentation/cassandra-operator-upgrade.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Cassandra Operator Upgrade
```

Comment:
As it is not "only" the operator, but the clusters too, naming the file and title just "Cassandra Upgrade".